### DISCUSSION TITLE
Outlined paper container & pagination shadows

### DISCUSSION THREAD

**Discussion by andokai** [KevinVandy/material-react-table#72] [General]
First of all, this library looks great. I look forward to the 1.0 release.

Now I'm not sure if this is a bug or a discussion so I'll start here.

The elevated paper container is useful in certain contexts but not very useful in many more. 

Would it be possible to not wrap this in a Paper component so that the elevation and padding etc. is an exercise for the user.

I see the `muiTablePaperProps` and that allows me to set the elevation and variant props however it appears the shadows have been applied manually to the pagination control so you end up with some weird shadows when you want a flat design.

https://codesandbox.io/s/strange-zhukovsky-yxhc5k?file=/src/TS.tsx

It also seems that the default paper border radius is removed which looks inconsistent. Normally there's a "square" prop you can use to enable/disable this but it doesn't appear to do anything.

**Comment by KevinVandy** [comment]
It was an original goal to be able to do what you are describing. I'll address the box shadow issue coming from the bottom toolbar

**Comment by KevinVandy** [comment]
This should be fixed in beta.10

Also, the recommended way to remove box shadow around the table is to simple set the elevation={0} 

```js
    muiTablePaperProps={{
      elevation: 0,
    }}
```

**Comment by andokai** [comment]
Hi, it's better but not quite right. It still appears to have the shadow on the left had side.

<img width="715" alt="image" src="https://user-images.githubusercontent.com/3984138/189444032-d4c96d41-f259-47a8-9afa-d43de6bb54a7.png">

Updated the sandbox at the link above to demonstrate. It's the bottom table of three.

**Reply by KevinVandy** [reply]
That sandbox is still on beta.9. I saved the package.json to force a re-install and it looks fixed on beta.10

<img width="1142" alt="image" src="https://user-images.githubusercontent.com/28243511/189451995-f7346a6d-dd7a-406c-9edb-eeb227e6301e.png">

**Reply by andokai** [reply]
Very odd, I had done the same. Anyway, looks good now, thanks!
